templates/repo/issue/view_content.tmpl

```diff
@@ -9,109 +9,69 @@
 	{{end}}
 
 	{{ $createdStr:= TimeSinceUnix .Issue.CreatedUnix $.Lang }}
-	<div class="twelve wide column comment-list">
+	<div class="twelve wide column comment-list prevent-before-timeline">
 		<ui class="ui comments">
-			<div class="comment">
-				<a class="avatar" {{if gt .Issue.Poster.ID 0}}href="{{.Issue.Poster.HomeLink}}"{{end}}>
-					<img src="{{.Issue.Poster.RelAvatarLink}}">
-				</a>
-				<div class="content">
-					<div class="ui top attached header">
-						<span class="text grey"><a {{if gt .Issue.Poster.ID 0}}href="{{.Issue.Poster.HomeLink}}"{{end}}>{{.Issue.Poster.GetDisplayName}}</a> {{.i18n.Tr "repo.issues.commented_at" .Issue.HashTag $createdStr | Safe}}</span>
-						{{if not $.Repository.IsArchived}}
-							<div class="ui right actions">
-								{{template "repo/issue/view_content/add_reaction" Dict "ctx" $ "ActionURL" (Printf "%s/issues/%d/reactions" $.RepoLink .Issue.Index) }}
-								{{if or .IsIssueWriter .IsIssuePoster}}
-									<div class="item action">
-										<a class="edit-content" href="#"><i class="octicon octicon-pencil"></i></a>
-									</div>
-								{{end}}
-							</div>
-						{{end}}
-					</div>
-					<div class="ui attached segment">
-						<div class="render-content markdown has-emoji">
-							{{if .Issue.RenderedContent}}
-								{{.Issue.RenderedContent|Str2html}}
-							{{else}}
-								<span class="no-content">{{.i18n.Tr "repo.issues.no_content"}}</span>
+			<div class="timeline-line">
```

Comment:
Instead of adding another wrapper div, would it instead be possible to add the class to the parent node?

Reply:
I will number the lines here, so i can easyly refer to them. Current structure is as follows:

    [1] <div class="twelve wide column comment-list prevent-before-timeline">
	[2]		<ui class="ui comments">
	[3]			<div class="timeline-line">
				... some Event Elements ...
	[4]			</div>
	[5]		</div>
	[6]	</div>
	
	
First ignore my changes and new added lines [3] and [4].

## Current Master

|![grafik](https://user-images.githubusercontent.com/24939127/60745641-9b7d8880-9f7b-11e9-91c6-df3390c00de8.png) | ![grafik](https://user-images.githubusercontent.com/24939127/60745758-f44d2100-9f7b-11e9-9b49-3f69b3acc1af.png) |
|--|--|

As you can see the box has at least the height of the sidebar which is the cause of the problem. The issue-timeline is a `before` element of [1], `position:absolut`, `100% height`.  If there are too less entries inside, the line overflows.

```
Solution: Move the line to an element with no height restriction, 
          which grows with the content.
```

## Move `:before` to UI container - [2]

| ![grafik](https://user-images.githubusercontent.com/24939127/60745853-6291e380-9f7c-11e9-83a3-2111a36c4923.png) | ![grafik](https://user-images.githubusercontent.com/24939127/60745861-6887c480-9f7c-11e9-9358-b0e064df241a.png) |
|--|--|


As you suggested I tried to move the line to the `ui` element. (I did this previously as well :) ) Additionally [2] has to get the rule `position:relative` so an absolute positioned element can access height of it.

| ![grafik](https://user-images.githubusercontent.com/24939127/60745967-d7fdb400-9f7c-11e9-9c27-44a994c6bf8b.png)  | ![grafik](https://user-images.githubusercontent.com/24939127/60745981-e77cfd00-9f7c-11e9-819d-679942524795.png)  |
|--|--|

Strangely the before element has zero height... The reason is: ui[2] is set to `display:inline`. Pure `Inline` elements haven't`got a height or width, no border nor padding. Now I tried to change this.

(I changed the background color so the line will be clearly visible)

| ![Peek 2019-07-05 23-07](https://user-images.githubusercontent.com/24939127/60746029-12675100-9f7d-11e9-8a6d-dc37c573fa14.gif) | ![grafik](https://user-images.githubusercontent.com/24939127/60745981-e77cfd00-9f7c-11e9-819d-679942524795.png)  |
|--|--|

**Unfortunately** all other options will break the UI. That's the reason why I added an additional container [3] witch grows automatically and is referable by `position` property.

Reply:
yeah, found a way ... was just too easy ;)
changed now